flocker/dockerplugin/_api.py

```diff
@@ -28,7 +28,7 @@
 from ..control._config import dataset_id_from_name
 from ..apiclient import DatasetAlreadyExists
 from ..node.agents.blockdevice import PROFILE_METADATA_KEY
-
+from .testtools import parse_num
```

Comment:
tx, normally i would put this in something like `utils` but havent seen that convention, i guess i can add it to `_api`, but seems odd.